DTS-STN/secure-client-hub
# Remove apollo and add persisted queries

## [ADO-155141](https://dev.azure.com/VP-BD/DECD/_workitems/edit/155141)

Fixed [AB#155141](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/155141)

### Changelog
feat:added support for persisted queries and removed apollo client as a dependency

### Description of proposed changes:
This PR removes Apollo client from our app and instead makes GET requests to the persisted query endpoint to retrieve data.

### What to test for/How to test
1. Pull in branch
2. Ensure that you have the new value for `AEM_GRAPHQL_ENDPOINT` set. If you don't have this, please message me.
3. Type `npm run dev`
4. Ensure the application runs as expected
